requirements/base.txt

```diff
@@ -11,4 +11,5 @@ docker>=3.3.0
 dateparser~=0.7
 python-dateutil~=2.6
 pathlib2~=2.3.2; python_version<"3.4"
-
+airspeed~=0.5.10
+jsonpath-rw~=1.4.0
```

Comment:
Doesn't look like 3.7 is supported and notes that there are segfaults in the README which is concerning: "It is tested with Python 2.6, 2.7, 3.2, 3.3, 3.4, 3.5, and 3.6. (On travis-ci there is a segfault when running the tests with pypy; I don't think the problem lies with this library)."

Reply:
Same comment as above with airspeed. The specific error occurring looks to be for pypy vs. the regular install of python. Unless a large number of users of SAM Local are using Pypy this seems like less of a concern. As for support for python 3.7, I have a PR in for that right now (see https://github.com/kennknowles/python-jsonpath-rw/pull/65)